### PR TITLE
Issue #138 [ci skip] adding setup info for windows

### PR DESCRIPTION
**Pull request by YipengHu**
# Description

doc updated to add windows setup info - win+gpu will not be supported

### CODE REVIEW

**Comment by YipengHu** [comment]
@NMontanaBrown if you want to add a another tab for Mac?

File: docs/quick_start.md [hunk 1]
```diff
@@ -33,6 +33,51 @@ conda create --name deepreg python=3.7 tensorflow-gpu=2.2 # use conda to handle
...
+
+in the Deepreg root directory. This will run unit tests and produce a coverage report of
+the local repository.
+
+#### ** Windows **
+
+With CPU only
+
+```
+[Windows Subsystem Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10) is recommended. Then follow the instructions with Linux.
```
Comment (mathpluscode) [inline docs/quick_start.md:69]: this line is not code, so not recommended to use ``` to wrap it

File: docs/quick_start.md [hunk 2]
```diff
@@ -33,6 +33,51 @@ conda create --name deepreg python=3.7 tensorflow-gpu=2.2 # use conda to handle
...
+```terminal
+conda create --name deepreg python=3.7 tensorflow-gpu=2.2 # use conda to handle nvidia related packages
+conda activate deepreg
+```
+
+- Regularly tested on Ubuntu/Debian. To check tests pass locally, run:
+
+```terminal
+conda activate deepreg
+pytest --cov-report=term --cov=deepreg test/unit/
```
Comment (mathpluscode) [inline docs/quick_start.md:58]: if i understand correctly, we want to let users to test their installations by running unit test,
then i recommend to move this part outside of the tab,
as this is not only for mac ;)

Comment (mathpluscode) [reply docs/quick_start.md:58]: because, it seems that the linux/mac/win tabs are not synced

File: docs/quick_start.md [hunk 3]
```diff
@@ -23,16 +23,47 @@ With CPU only
...
+With CPU only
+
+_[Windows Subsystem Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10)
+is recommended. Then follow the instructions with Linux._
+
+With GPU
+
+:warning: Not supported or tested.
+
+- Regularly tested on WSL 1/2.
```
Comment (mathpluscode) [inline docs/quick_start.md:65]: maybe not this line?

**Pull request merged**